packages/location-management/src/utils.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Find the pivot/mid index of an array
+ *
+ * @param {number[] | string[]} array Array of ParsedHierarchyNode
+ * @param {number} start stating index to look for pivot
+ * @returns {number} returns the pivot/mid index of an array
+ */
+function pivot(array: number[] | string[], start = 0) {
+  const pivot = array[start];
+  let pointer = start;
+
+  for (let i = start; i < array.length; i++) {
+    if (array[i] < pivot) {
+      pointer++;
+      [array[pointer], array[i]] = [array[i], array[pointer]];
+    }
+  }
+
+  if (pointer < array.length) [array[start], array[pointer]] = [array[pointer], array[start]];
+
+  return pointer;
+}
+
+/**
+ * Sort the Base ParsedHierarchyNode by name
+ *
+ * @param {number[] | string[]} array Array of ParsedHierarchyNode
```

Comment:
looks like the type and the docstring are mismatched here

Reply:
Resolved